plugins/modules/keycloak_clientscope_type.py

```diff
@@ -190,6 +190,15 @@ def extract_field(dictionary, field='name'):
     return [cs[field] for cs in dictionary]
 
 
+def normalize_scopes(scopes):
+    scopes_copy = scopes.copy()
+    if isinstance(scopes_copy.get('default_clientscopes'), list):
+        scopes_copy['default_clientscopes'] = sorted(scopes_copy['default_clientscopes'])
+    if isinstance(scopes_copy.get('optional_clientscopes'), list):
+        scopes_copy['optional_clientscopes'] = sorted(scopes_copy['optional_clientscopes'])
```

Comment:
Doing the copy is a bit redundant. This could be simplified as:
```suggestion
    scopes_copy = {
        'default_clientscopes': sorted(scopes['default_clientscopes']) if scopes['default_clientscopes'] else [],
        'optional_clientscopes': sorted(scopes['optional_clientscopes']) if scopes['optional_clientscopes'] else [], 
    }
```